cvat-canvas3d/src/typescript/cuboid.ts

```diff
@@ -115,6 +140,39 @@ export class CuboidModel {
         this.front.add(camRotateHelper.clone());
     }
 
+    private createArrows(): ObjectArrowHelper {
+        return {
+            x: new THREE.ArrowHelper(new THREE.Vector3(1, 0, 0), new THREE.Vector3(0.5, 0, 0), 2, 0xff0000),
+            y: new THREE.ArrowHelper(new THREE.Vector3(0, 1, 0), new THREE.Vector3(0, 0.5, 0), 2, 0x00ff00),
+            z: new THREE.ArrowHelper(new THREE.Vector3(0, 0, 1), new THREE.Vector3(0, 0, 0.5), 2, 0x0000ff),
+        };
+    }
+
+    public updateArrowLengths(): void {
```

Comment:
perhaps you need to update only arrowOrigin (second argument in THREE.ArrowHelper constructor), not arrowLength